neuralmonkey/evaluators/chrf.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Optional
+
+
+# pylint: disable=too-few-public-methods
+class ChrFEvaluator(object):
+    """Compute ChrF introduced in
+    http://www.statmt.org/wmt15/pdf/WMT49.pdf
+    """
+
+    def __init__(self, n: int = 3, beta: float = 1,
+                 name: Optional[str] = None) -> None:
+        self.n = n
+        # We store the squared value of Beta
+        self.beta_2 = beta**2
```

Comment:
Aha, tak takhle to je správně podle PEP 8 protože to je operátor s vysokou prioritou..